tests/graphql/test-graphql-api.php

```diff
@@ -0,0 +1,135 @@
+<?php
+/**
+ * Class GraphQLAPITest
+ *
+ * @package vip-block-data-api
+ */
+
+namespace WPCOMVIP\BlockDataApi;
+
+/**
+ * Tests for the GraphQL API.
+ */
+class GraphQLAPITest extends RegistryTestCase {
+
+	public function test_is_graphql_enabled_true() {
+		$this->assertTrue( apply_filters( 'vip_block_data_api__is_graphql_enabled', true ) );
+	}
+
+	public function test_is_graphql_enabled_false() {
+		$is_graphql_enabled_function = function () {
+			return false;
+		};
+		add_filter( 'vip_block_data_api__is_graphql_enabled', $is_graphql_enabled_function, 10, 0 );
+		$this->assertFalse( apply_filters( 'vip_block_data_api__is_graphql_enabled', true ) );
+		remove_filter( 'vip_block_data_api__is_graphql_enabled', $is_graphql_enabled_function, 10, 0 );
+	}
+
+	public function test_get_blocks_data() {
```

Comment:
This one test battle tests the entire graphql code. We can test the exact GraphQL query as we need wp-graphQL so this is a good middle ground for that.